notebooks/viz/plot-metric-correlation.ipynb

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
Seems like the quicklook has a lot of markdown. Would this notebook benefit from some markdown w/ context for the use cases and additional info?

Reply:
I'll address this in #161